src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,132 @@
+---
+title: "2023 Web Framework Performance Report"
+description: "A look at how different web frameworks perform in the real world in 2023. Based on real-world, production data from HTTP Archive and Google Chrome."
+publishDate: "March 7, 2023"
+authors:
+  - fred
+lang: "en"
+socialImage: "/src/content/blog/_images/web-framework-performance-report/social.jpg"
+---
+
+import CWVCulminativeChart from "/src/components/charts/CWVCulminativeChart.astro"
+import CWVFIDChart from "/src/components/charts/CWVFIDChart.astro"
+import CWVLCPChart from "/src/components/charts/CWVLCPChart.astro"
+import CWVCLSChart from "/src/components/charts/CWVCLSChart.astro"
+import CWVINPChart from "/src/components/charts/CWVINPChart.astro"
+import LighthouseMedianChart from "/src/components/charts/LighthouseMedianChart.astro"
+import LighthousePercentileChart from "/src/components/charts/LighthousePercentileChart.astro"
+import CostOfJSChart from "/src/components/charts/CostOfJSChart.astro"
+
+[INTRO]
+
+# The Data
+
+The purpose of this report is to look at real-world data to better understand the relationship between framework choice, performance, and actual user experience on the web. To do this, we looked at three different third-party datasets:
+
+- [The Chrome User Experience Report (CrUX)](https://developer.chrome.com/docs/crux/) provides user experience metrics for how real-world Chrome users experience popular destinations on the web.
+- [The HTTP Archive](https://httparchive.org/) which tracks and reports the performance of over 15 million websites over time by regularly collecting Lighthouse performance data.
+- [The Core Web Vitals Technology Report](https://discuss.httparchive.org/t/new-dashboard-the-core-web-vitals-technology-report/2178) which pulls insights from the previous two datasets.
```

Comment:
Did we actually look at three different data sources, or did we look at the `The Core Web Vitals Technology Report` which combines the data from `CrUX` and `HTTP Archive`?

Reply:
Only the second two were queried directly, I'll make a note of that